.github/workflows/kitchen.yml

```diff
@@ -56,7 +56,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-10.15] # macos-11.0 is not public for now
+        os: [macos-11] # Github has deprecated macos-10.15 runner https://github.com/actions/runner-images/issues/5583
```

Comment:
why not bump to macos-12

Reply:
Let me try if we get any unexpected issues with macos12. 
Currently working on getting a working release pipeline for chef16 as the previous artifacts have expired, so the kitchen tests fail with 404